### PR TITLE
LMU-147/almost done

### PR DESCRIPTION
**Pull request by Aminebouafia**

### CODE REVIEW

File: src/components/Dashboard/BetaContainer.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    >
+      <Box className={classes.boxkbir} mt="20px">
+        <Box
+          className={classes.boxloul}
+        >
+
+          <Box
+            display="flex"
+          >
+            <UserAvatar size="49px" src={getFile(data.cover_photo)} />
```
Comment (Nabil-MEDIOUNI) [inline src/components/Dashboard/BetaContainer.js:41]: data is missing currentPerson: data.currentPerson.cover_photo

File: src/components/Dashboard/BetaContainer.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      width="100%"
+    >
+      <Box className={classes.boxkbir} mt="20px">
+        <Box
+          className={classes.boxloul}
+        >
+
+          <Box
+            display="flex"
+          >
```
Comment (Nabil-MEDIOUNI) [inline src/components/Dashboard/BetaContainer.js:40]: add alignItems="center" to center the name and the avatar))

File: src/components/Dashboard/BetaContainer.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+            </Box>
+          </Box>
+          <Box display="flex">
+            <Typography variant="overline" className={classes.profile} onClick={() => location.push('/settings/profile')}>
+              View Profile
+            </Typography>
+          </Box>
+        </Box>
+        <Divider />
+        <Box className={classes.boxtheny}>
```
Comment (Nabil-MEDIOUNI) [inline src/components/Dashboard/BetaContainer.js:58]: please avoid "ederja" in writing code))

File: src/components/Dashboard/BetaContainer.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+          <Box mb="30px">
+            <Typography>
+              Quick links
+            </Typography>
+          </Box>
+          <Box className={classes.boxlywestou}>
```
Comment (Nabil-MEDIOUNI) [inline src/components/Dashboard/BetaContainer.js:64]: same here

**Review by Nabil-MEDIOUNI — CHANGES_REQUESTED** [review]
please comment the quick links below, we won't need them.
increase the font-size of the quick links and decrease the size of images.
add some margin-top  for the first box and fix the positioning of the word quick links and size.

File: src/components/Dashboard/BetaContainer.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+  const { data, loading } = useContext(UserInfoContext);
+  const nonOperationFunctions = ['MKT', 'F&L', 'PR', 'TM', 'IM'];
+  const FunctionName = data.currentPerson.current_positions.function.function_short_name;
+
+  return (
+
+    <div
+      width="100%"
+    >
+      <Box className={classes.boxkbir} mt="20px">
```
Comment (Nabil-MEDIOUNI) [inline src/components/Dashboard/BetaContainer.js:33]: add a <Box /> here for gray bg
<Box position="fixed" bgcolor="#F2F3F6" width="100%" height="100vh" zIndex={-1} />

**Comment by amalwsl** [comment]
good job <3

**Pull request merged**